cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -193,7 +195,7 @@ def update(self, updates, cancel=False, reset=False):
             })
             if self.appliance.version < '5.9':
                 fill_dict['depot_creds']['confirm_password'] = updates.get('password')
-        updated = view.fill(fill_dict)
+        updated = view.fill(fill_dict) or updated
```

Comment:
If `view.fill()` is false here, and depot_type was not 'Red Hat Dropbox' above (empty fill dict?) this will cause UnboundLocal as it'll try and use `updated` to set `updated`